libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ProbabilityDistributionChart.tsx

```diff
@@ -72,19 +70,9 @@ export class ProbabilityDistributionChart extends React.Component<
     }
     const theme = getTheme();
 
-    const selectedDatasetCohorts = this.props.datasetCohorts.filter(
-      (_cohort, index) => {
-        return this.props.selectedDatasetCohorts.includes(index);
-      }
-    );
-    const selectedFeatureBasedCohorts = this.props.featureBasedCohorts.filter(
-      (_cohort, index) => {
-        return this.props.selectedFeatureBasedCohorts.includes(index);
-      }
-    );
-    const selectedCohorts = selectedDatasetCohorts.concat(
-      selectedFeatureBasedCohorts
-    );
+    const selectedCohorts = this.props.cohorts.filter((_cohort, index) => {
```

Comment:
This is o(n^2) ^_^
Seems fine since # of cohort is limited.

Reply:
see my response below https://github.com/microsoft/responsible-ai-toolbox/pull/1375#issuecomment-1111512174